homeassistant/components/conversation/default_agent.py

```diff
@@ -788,11 +791,25 @@ def _make_slot_lists(self) -> dict[str, SlotList]:
 
                     area_names.append((alias, area.id))
 
-        _LOGGER.debug("Exposed entities: %s", entity_names)
+        # Expose all floors.
+        #
+        # We pass in floor id here with the expectation that no two areas will
+        # share the same name or alias.
+        floors = fr.async_get(self.hass)
+        floor_names = []
+        for floor in floors.async_list_floors():
+            floor_names.append((floor.name, floor.floor_id))
+            if floor.aliases:
+                for alias in floor.aliases:
+                    if not alias.strip():
+                        continue
+
+                    floor_names.append((alias, floor.floor_id))
 
         self._slot_lists = {
             "area": TextSlotList.from_tuples(area_names, allow_template=False),
             "name": TextSlotList.from_tuples(entity_names, allow_template=False),
+            "floor": TextSlotList.from_tuples(floor_names, allow_template=False),
```

Comment:
Should we listen to floor registry changed to clear the slot lists like we do for area?

https://github.com/home-assistant/core/blob/a2039ff1111a987069416875f2d615d05284fc65/homeassistant/components/conversation/default_agent.py#L165-L169

Reply:
Yes! I missed that.